components/blitz/resources/omero/cmd/FS.ice

```diff
@@ -185,41 +185,6 @@ module omero {
 
         };
 
-        /**
-         * Request to determine the disk usage of the given objects
-         * and their contents. File-system paths used by multiple objects
-         * are de-duplicated in the total count. Specifying a class is
-         * equivalent to specifying all its instances as objects.
-         *
-         * Permissible classes include:
-         *   ExperimenterGroup, Experimenter, Project, Dataset,
-         *   Folder, Screen, Plate, Well, WellSample,
-         *   Image, Pixels, Annotation, Job, Fileset, OriginalFile.
-         **/
-        class DiskUsage extends Request {
```

Comment:
I was thinking that the API has changed very little; indeed, the policy rules should pretty much mirror what the previous did. Should I just rename it or also resurrect the previous as deprecated? (I suppose they could coexist, though of course this diff will then do even worse at showing the changes!)